README.md

```diff
@@ -90,11 +104,11 @@ WSGIPythonPath usr/local/devcycle
 * Change the application settings:
 
 ```
-nano /usr/local/devcycle/dataCollection/settings.py
+sudo vim /usr/local/devcycle/dataCollection/settings.py
```

Comment:
well I didn't tell them to be root and they won't have the permissions for /usr/local or /etc/apache2